static/app/utils/gettingStartedDocs/node.ts

```diff
@@ -144,6 +144,12 @@ node --require ./instrument.js app.js
 node --import ./instrument.mjs app.mjs`;
 }
 
+export function getImportInstrumentSnippet(): string {
+  return `// IMPORTANT: Make sure to import \`instrument.js'\` at the top of your file.
+  // If you're using ECMAScript Modules (ESM), use \`import './instrument.js';\`
+  require('./instrument.js');`;
```

Comment:
We maybe can improve on this, but for now that's OK - let's ship this and tweak it later!